connector/spanmetricsconnectorv2/testdata/with_custom_histogram_buckets/config.yaml

```diff
@@ -0,0 +1,8 @@
+spanmetricsv2:
+  spans:
+    trace.span.duration:
+      description: Span duration with custom histogram buckets
+      unit: "s"
+      histogram:
+        explicit:
+          buckets: [0.001, 0.1, 10, 100]
```

Comment:
Maybe alter the buckets so one of the span durations is greater than the greatest bucket value? i.e. to cover the case where SearchFloat64s returns len(buckets)

Reply:
Done!